server/app/routes/api/reviews.js

```diff
@@ -1,20 +1,69 @@
 var express = require('express');
-var router = express.Router();
+var router = express.Router({ mergeParams: true });
 var mongoose = require('mongoose');
 
+require('../../../db/models/review');
+var Review = mongoose.model('Review');
+
+
 // available to eveyone
-router.get('/:id/reviews', function (req, res) {
+router.get('/', function (req, res, next) {
   var id = req.params.id;
-  mongoose.model('User')
-  .findById(id)
-  .then(function(user) {
-    return user.findReviews()
-  })
+  Review.find({author: id})
   .then(function(reviews) {
     res.json(reviews)
   })
-  .then(null, function(err) {
-    console.log(err);
-    res.sendStatus(404);
+  .then(null, next)
+})
+
+router.post('/', function(req, res, next) {
+  Review.create(req.body)
+  .then(function(review){
+    if (review.author === req.params.id) {
+      res.json(review);
+    } else {
```

Comment:
It might be a good idea to have multiple layers on security? Not sure.
But other users should be able to see each others reviews, I would think. Maybe on the user's "profile" page or something?